<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WelcomeInfo</class>
 <widget class="QDialog" name="WelcomeInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>811</width>
    <height>802</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../huggle_res/pictures.qrc">
    <normaloff>:/huggle/pictures/Resources/huggle3_small.ico</normaloff>:/huggle/pictures/Resources/huggle3_small.ico</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>36</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Welcome to Huggle!</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Language</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cb_Language">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;It looks like this is the first time you have launched Huggle!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblWelcomeText">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Welcome! Here are some quick tips that will help you out while using this program:&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Logging in&lt;/span&gt;&lt;/p&gt;&lt;p&gt;We support 2 types of login. The first is called &amp;quot;bot login&amp;quot;, but this can also be considered as an application login, as it serves the API for third-party applications. This is the preferred way to log in as it is generally more secure. In order to use bot login, you first need to generate a login password. In order to do that, visit [[Special:BotPasswords]] page on your wiki. That will give you a new special password that you will supply to Huggle instead of normal one.&lt;/p&gt;&lt;p&gt;The other supported login is a standard one, where you can use the same username and password as you would when logging in to a wiki. Support for this login may be dropped by MediaWiki in future versions.&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Using Huggle&lt;/span&gt;&lt;/p&gt;&lt;p&gt;The long and detailed manual is here: &lt;a href=&quot;https://www.mediawiki.org/wiki/Manual:Huggle&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;https://www.mediawiki.org/wiki/Manual:Huggle&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;p&gt;For these lazy to read it, here are few tips to get started really quickly:&lt;/p&gt;&lt;p&gt;* Huggle is a diff browser, that means it will connect to a recent changes stream and load wiki diffs into its memory. You will see pre-loaded diffs on left side in window called &amp;quot;Queue&amp;quot;&lt;/p&gt;&lt;p&gt;* Navigate to next edit in a Queue by using spacebar or click next button&lt;/p&gt;&lt;p&gt;* To move back and forward between diffs you have opened, use the square brackets as a shortcut [ ]&lt;/p&gt;&lt;p&gt;* To revert a harmful edit and warn the user who made it, press Q. If you want to just revert it without sending a message, press R.&lt;/p&gt;&lt;p&gt;* You can also pick more specific reasons and warning messages for reverting a change in the dropdown menu in the toolbar&lt;/p&gt;&lt;p&gt;* Huggle allows you to revert edits, restore specific revisions, send warnings to users, welcome users, tag pages, delete pages, view user information, block users and many other advanced features.&lt;/p&gt;&lt;p&gt;* Be careful. This tool can blow the wiki up. Make sure you are familiar with dealing with vandalism on the wiki before you start reverting. Don't be too fast, take breaks. Think twice before pressing Q (or R). If you made a mistake, press ESC ASAP and you might be able to cancel the revert before the change goes live. If you were too slow, right click the revert in history window (bottom left) and click Undo. That will restore your revert and undo warnings sent to user.&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Getting help&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Best way to get help is probably visiting the feedback page on the English Wikipedia at &lt;a href=&quot;https://en.wikipedia.org/wiki/WP:HG/T&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;https://en.wikipedia.org/wiki/WP:Huggle/Feedback&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Clicking OK will get you to login page. Pick up a project(s) you wish to login to, enter your credentials and login.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox">
     <property name="text">
      <string>Show this message next time</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnOK">
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../huggle_res/pictures.qrc"/>
 </resources>
 <connections/>
</ui>
